apps/openssl.c

```diff
@@ -29,6 +29,7 @@
 #endif
 #define INCLUDE_FUNCTION_TABLE
```

Comment:
This only affects progs.h so while you're here, move the define.

Reply:
I'm not sure I understand where you think it should be moved...

Reply:
Oh wait, just down one line?  Yeah I can do that